src/js/polyfills/webvtt.js

```diff
@@ -551,6 +557,11 @@ function BoxPosition(obj) {
     this.width = obj.width || width;
     this.lineHeight = lh !== undefined ? lh : obj.lineHeight;
 
+    if (cueDiv && div && this.width > div.parentElement.offsetWidth) {
```

Comment:
since we are wrapping the text, shouldn't we also set the width to the parent width?

Reply:
moved this logic out of BoxPosition